desktop-exporter/app/utils/duration.ts

```diff
@@ -1,28 +1,55 @@
 import Timestamp from "timestamp-nano";
 import { SpanData } from "../types/api-types";
 
-export function getSpanDuration(startTimestamp: string, endTimestamp: string) {
-  try {
-    let startTimeMs = Date.parse(startTimestamp.split(".")[0]);
-    let endTimeMs = Date.parse(endTimestamp.split(".")[0]);
-
-    let startTimeNs =
-      startTimeMs * 1e6 + Timestamp.fromString(startTimestamp).getNano();
-    let endTimeNs =
-      endTimeMs * 1e6 + Timestamp.fromString(endTimestamp).getNano();
-
-    return endTimeNs - startTimeNs;
-  } catch (error) {
-    return null;
+export type TraceTimeAttributes = {
+  traceStartTimeNS: number;
+  traceDurationNS: number;
+};
+
+export function getTraceTimeAttributes(spans: SpanData[]): TraceTimeAttributes {
```

Comment:
This name is a bit odd. `calculateTraceDuration` maybe?

Reply:
I went with calculateTraceTiming because I need the start and the duration.